serve-react-app/Makefile

```diff
@@ -0,0 +1,13 @@
+assets := $(wildcard hello/*)
+
+run: $(assets)
+	cd hello/assets && npm run build -- --watch&
```

Comment:
```suggestion
	npm --prefix hello/assets run build -- --watch&
```

Reply:
10 years of JavaScript and I never knew `--prefix` was a thing

Reply:
It's one of the joys of coding 😄